tests/unit/s2n_self_talk_ktls_test.c

```diff
@@ -698,5 +683,93 @@ int main(int argc, char **argv)
         }
     }
 
+    /* Test: Keyupdates with KTLS */
+    if (ktls_keyupdate_expected) {
+        /* Test: Sending key update with KTLS */
```

Comment:
Can we make these tests end-to-end? I'm worried that the tests in s2n_ktls_io_test only test half the process, and these tests only test the other half of the process, and we don't have any tests that test the entire process.